src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -569,6 +572,7 @@ def load_arguments(self, _):
     with self.argument_context('ts update') as c:
         c.argument('resource_group', arg_type=resource_group_name_type, help='The resource group to store the template spec.')
         c.argument('template_spec', arg_type=deployment_template_spec_type)
+        c.argument('ui_form_definition_file', arg_type=ui_form_definition_file_type, help='The uiFormDefinition file path in the file system for the template spec version.')
```

Comment:
For `ts create`, the help info is `... template spec`, but for `ts update`, the help info is `... template spec version`
May I ask why is this help info different from the help in `ts create`?

Reply:
Will change both to template spec version.